lineman/app/components/navbar/navbar.scss

```diff
@@ -22,8 +33,10 @@
 
 .lmo-navbar__item{
   display: inline-block;
-  margin: 0 0 0 2px;
-  height: $navbar-height - 4px;
+  margin: 0;
+  .navbar-user-options .lmo-navbar__btn {
```

Comment:
Maybe also have a look at https://css-tricks.com/bem-101/ if you like. Let me know if either of those seem better than the other. I want to include a good link in the Code Review Checklist/Style guide